### ISSUE TITLE
RPI4 aarch64 kernel 5.15 7_45_185 or 206 packet injection doesn't work

### ISSUE THREAD

**Issue by MakcuK-Mipt** [seemoo-lab/nexmon#557]
Hi
Patched without errors. Was able to switch wlan0 to monitor mode. Can scan networks by airodump-ng.
But aireplay-ng -9 wlan0mon showing bad result on NO packet injection:
22:42:33  Trying broadcast probe requests...
22:42:34  No Answer...
22:42:34  Found 1 AP

22:42:34  Trying directed probe requests...
22:42:34  74:4D:28:9D:67:AC - channel: 3 - 'MHome2G'
22:42:40   0/30:   0%
Does it work with RPI4 aarch64 kernel 5.15? Or maybe it cannot work with aircrack?
Please advice!
Regards.

**Comment by jayofelony** [comment]
Yes, I'm seeing this issue as well with 7_45_206 installed on RPi4 8GB. I tried reinstalling brcmfmac drivers which succeeded, but with aireplay-ng --test mon0, and wlan0mon I get no answer.
